core/templates/services/questions-list.service.ts

```diff
@@ -34,7 +34,8 @@ import { TruncatePipe } from 'filters/string-utility-filters/truncate.pipe';
 })
 export class QuestionsListService {
   private _questionSummariesForOneSkill: QuestionSummaryForOneSkill[] = [];
-  private _nextCursorForQuestions: string = '';
+  private _nextOffsetForQuestions: number = 0;
+  private _moreQuestions: boolean = true;
```

Comment:
This is a bit unclear. Maybe _moreQuestionsAvailable, and add a line comment above it saying "Whether there are more questions available to fetch."

(Please make the corresponding name changes below.)